.eslintrc

```diff
@@ -450,6 +458,7 @@
         "packages/dashboard/src/karma-tests.cjs",
         "packages/story-editor/src/karma-tests.cjs",
         "packages/migration/src/**/*.js",
+        "packages/eslint-import-resolver/src/*.js",
```

Comment:
Not sure why this doesn't have any effect 🤔

Reply:
Just guessing that maybe because it's `*.js` instead of `*.cjs` and `"type": "module"` is defined in `package.json` so it's considering it es module using `require` ?